todos-app/backend/task_service/spec/rails_helper.rb

```diff
@@ -8,7 +8,7 @@
 abort('The Rails environment is running in production mode!') if Rails.env.production?
 require 'rspec/rails'
 # Add additional requires below this line. Rails is not loaded until this point!
-
+Rails.application.load_tasks
```

Comment:
Hmm. I never saw this line in rails_helper.rb inside other rails project.
Most of them is putting this line inside a `project/Rakefile`.

Is this auto generated of you added it?

Reply:
This line is added for exclusively for rspec tests. If I don't have this line, rspec will complain that it doesn't know about Rake tasks like this
```bash
Failure/Error: Rake::Task['maintenance:end'].execute

NameError:
  uninitialized constant Rake::Task
```
I only need to run that line once, so I thought about where to put it. I thought `rails_helper` is a good enough option.

Reply:
ohh... In that case it's fine! Thanks for explaining.